example/command_line_example/osm_test.md

```diff
@@ -0,0 +1,42 @@
+## Example to test implementation for reading OSM data directly
+
+These instructions were tested on Ubuntu 18.04, after building the code in the OSM branch. They assume a working directory under ``example/command_line_example/``
+
+### 1. Download test data
+
+```
+mkdir osm_test
+cd osm_test
+wget https://www.dropbox.com/s/a3d8hmtkflckw10/fmm_osm_test_data.zip
+unzip fmm_osm_test_data.zip
+```
+
+Test data contains:
+ - OSM data for Singapore (defined by bounding box)
+ - Road network in shapefile format (converted from OSM with https://github.com/dkondor/osmconvert/ )
```

Comment:
As osmconvert is already used for this tool https://wiki.openstreetmap.org/wiki/Osmconvert.
Creating a routable road network from OSM is always a challenging problem. I suggest that maybe you can create a more powerful tool called osm2network which addresses the problems we have discussed. This is just a suggestion.

Reply:
Thanks for noting this, I'll rename my code to distinguish -- of course it is a work in progress, so far I use it mainly to create TSV format networks, and will add features as I need them :)